schematic/models/metadata.py

```diff
@@ -292,13 +298,13 @@ def submit_metadata_manifest(
         dataset_id: str,
         manifest_record_type: str,
         restrict_rules: bool,
-        validate_component: str = None,
+        access_token: str,
```

Comment:
should this one be `Optional[str] = None` because it used to be `str = None`?

Reply:
Ahh right, I think I had thought this was the validation method. thanks for clarifying

Reply:
@andrewelamb @GiaJordan We need to keep the None default, since this change actually broke the CLI submit if you require the access token. The CLI gets the access token another way from the API. I will submit another PR to update typing `Optional[str]=None`